parlai/tasks/eli5/data_creation/download_reddit_qalist.py

```diff
@@ -0,0 +1,539 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Adapted from https://github.com/facebookresearch/ELI5/blob/master/data_creation/download
```

Comment:
how much of this is copied?

Reply:
I added download_and_process_comments, download_and_process_posts, get_posts, get_comments_from_post, and a few parts of the parsing part main for downloading specific reddit posts. Rest is from ELI5.